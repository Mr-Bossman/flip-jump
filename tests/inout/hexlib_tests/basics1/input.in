0123456789:;<=>?
0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`abcdefghijklmno
0123456789:;<=>?
Flip, Jump!
