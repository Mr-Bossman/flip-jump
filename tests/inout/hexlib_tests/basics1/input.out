input_hex:
fedcba9876543210
0123456789XXXXXXXabcdefXXXXXXXXXXXXXXXXXXXXXXXXXXabcdefXXXXXXXXX

input-print:
0123456789:;<=>?
Flip, Jump!

